src/diffusers/models/mel.py

```diff
@@ -0,0 +1,233 @@
+# Copyright 2022 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
```

Comment:
This seeems a bit too specific at the moment to go into `src/diffusers/models` => could we maybe instead put it directly in `pipelines/audio_diffusion` like we're doing here: https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines/alt_diffusion/modeling_roberta_series.py

Reply:
Ok, I'll look into that. So that I understand, Mel would not be a composable module in the pipeline but I would create some specific config for the pipeline which would be used to instantiate the Mel object - is that right @patrickvonplaten ?

Reply:
Done.